packages/api/src/controllers/stream.ts

```diff
@@ -482,9 +482,7 @@ app.get("/:parentId/sessions", authorizer({}), async (req, res) => {
   if (record) {
     if (record === "true" || record === "1") {
       query.push(sql`data->>'record' = 'true'`);
-      query.push(sql`data->>'recordObjectStoreId' IS NOT NULL`);
     } else if (record === "false" || record === "0") {
-      query.push(sql`data->>'recordObjectStoreId' IS NULL`);
       filterOut = true;
```

Comment:
Might be missing sth like this?
```suggestion
      query.push(sql`(data->>'record' = 'false'` OR data->>'record' IS NULL)`);
      filterOut = true;
```

Reply:
not sure if `record` can ever be null, added it just to be safe